app/controllers/data_access_controller.rb

```diff
@@ -192,6 +197,17 @@ def options
 
   private
 
+  def echo_form_outdated?(access_config, methods)
+    return true if access_config.nil? || access_config.echoform_digest.nil? || methods.nil?
```

Comment:
What if there is no echoform, i.e. if the user had previously selected "Download", one of the OPeNDAP methods, or an order without a form?